spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -233,6 +233,21 @@ configuration.classes=org.springframework.cloud.stream.app.file.sink.FileSinkPro
 configuration-properties.names=server.port
 ```
 
+=== Global application properties (for streams)
+
+In addition to configuration via DSL, Spring Cloud Data Flow provides a mechanism for setting specific properties to all the streaming applications that are launched by it.
+This can be done by adding properties prefixed with `spring.cloud.dataflow.applicationProperties.stream` when starting the server.
+When doing so, the server will pass all the properties, without the prefix, to the instances it launches.
+
+For example, all the launched applications can be reconfigured to use a different Kafka broker by the default by launching the configuration server with the following options:
+
+```
+--spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.brokers=192.168.1.100:9092
+--spring.cloud.dataflow.applicationProperties.stream.spring.cloud.stream.kafka.binder.zkNodes=192.168.1.100:2181
+```.
+
+This will cause the properties `stream.spring.cloud.stream.kafka.binder.brokers` and `spring.cloud.stream.kafka.binder.zkNodes` to be passed to all the launched applications.
```

Comment:
Should mention here that IF the same property is provided explicitly with the `app.<name>.` prefix, that value would take precedence. And, on that note, would we possibly want a boolean flag to control that precedence? (thinking of the case where config-server might be used, and overriding explicit property values might be desirable).